lib/taskana-core/src/main/java/pro/taskana/TaskanaEngineConfiguration.java

```diff
@@ -135,6 +135,8 @@ public TaskanaEngineConfiguration(
       this.rolesSeparator = rolesSeparator;
     }
 
+    setGermanPublicHolidaysEnabled(true);
```

Comment:
This should obviously be configurable, shouldn't it? Otherwise the switch does not make sense.

Reply:
The setGermanPublicHolidaysEnabled() method is public.It is the interface, the customer can use to configure the TaskanaEngineConfiguration with respect to german holidays.

The referenced call simply sets the default to true. Without this call, the default is false.
Since the product is developed and mostly used in Germany, I think that **true** is the correct default...